UnicornRender/video/source/vulkan/Renderer.cpp

```diff
@@ -1313,11 +1278,11 @@ bool Renderer::CreateCommandBuffers()
                 if(pVkMesh->IsValid())
                 {
                     glm::vec4 colorPush(
-                        pVkMesh->GetColor(), // xyz - color
-                        pVkMesh->IsColored() // w - boolean flag for 1 enabled color or 0 disabled color
+                        pVkMesh->GetMesh().GetMaterial()->GetColor(), // xyz - color
```

Comment:
Considering how frequently you call `pVkMesh->GetMesh().GetMaterial()` I think it makes sense to store it as a local variable on relevant scope level.

Reply:
Okay